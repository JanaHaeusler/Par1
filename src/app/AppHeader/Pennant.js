import styled from 'styled-components/macro'
import { LogoIcon } from '../assets/Icons/Icons'

export default function Pennant() {

  return (
      <PennantStyled>
        <LogoIconStyled/>
      </PennantStyled>
  )
}

const PennantStyled = styled.div`
  display: flex; 
  justify-content: center;
  width: 60px;
  height: 60px;
  box-shadow: 0 0 10px var(--primary-dark);
  border-radius: 0 0 50px 50px;
  background-color: var(--text-light);
`
const LogoIconStyled = styled(LogoIcon)`
  margin-top: 10px;
  width: 30px;
  fill: var(--secondary-dark);
`