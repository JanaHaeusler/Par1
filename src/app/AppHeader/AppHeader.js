import styled from 'styled-components/macro'
import Pennant from './Pennant'

export default function AppHeader({ className }) {
  return (
    <Header className={className}>
      <PennantWrapper>
        <Pennant />
      </PennantWrapper>
    </Header>
  )
}

const Header = styled.header`
  position: relative;
  height: 50px;
  max-width: 1240px;
  margin: 0 auto;
  box-shadow: var(--shadow-dark);
  border-radius: 0 0 25px 3px;
  background: var(--primary-gradient);
`
const PennantWrapper = styled.div`
  position: absolute;
  top: 0;
  left: 28px;
`
