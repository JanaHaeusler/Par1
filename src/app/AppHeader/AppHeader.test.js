import AppHeader from './AppHeader'
import { render } from '@testing-library/react'

describe('AppHeader', () => {
    it('renders correctly', () => {
        const { container } = render(<AppHeader/>)
        expect(container.firstChild).toMatchSnapshot()
    })
})