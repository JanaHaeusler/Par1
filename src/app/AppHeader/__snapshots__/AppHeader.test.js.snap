// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader renders correctly 1`] = `
.c2 {
  height: 60px;
  width: 60px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  box-shadow: 0 0 5px var(--secondary-medium);
  border-radius: 0 0 50px 50px;
  background-color: var(--light);
}

.c3 {
  width: 30px;
  margin-top: 10px;
  fill: var(--primary);
}

.c0 {
  position: relative;
  height: 50px;
  max-width: 1240px;
  margin: 0 auto;
  box-shadow: 0 0 5px var(--secondary-medium);
  border-radius: 0 0 25px 3px;
  background: var(--primary-gradient);
}

.c1 {
  position: absolute;
  top: 0;
  left: 28px;
}

<header
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="c2"
    >
      <svg
        class="c3"
      >
        golf-logo.svg
      </svg>
    </div>
  </div>
</header>
`;
